docs/admin/multi-cluster.md

```diff
@@ -80,8 +80,9 @@ Second, decide how many clusters should be able to be unavailable at the same ti
 the number that can be unavailable `U`.  If you are not sure, then 1 is a fine choice.
 
 If it is allowable for load-balancing to direct traffic to any region in the event of a cluster failure, then
-you need `R + U` clusters.  If it is not (e.g you want to ensure low latency for all users in the event of a
-cluster failure), then you need to have `R * U` clusters (`U` in each of `R` regions).  In any case, try to put each cluster in a different zone.
+you need at least `min(R, U+1)` clusters.  If it is not (e.g you want to ensure low latency for all users
```

Comment:
Understood, but even then, min(R, U+1) is not correct, as far as I understand, even in your concrete example (4 regions, resilience to one zone outage)?  If R=4, and U=1, your equation gives an answer of 2, which is clearly not what you're after here, right?